engines/wintermute/utils/path_util.cpp

```diff
@@ -74,6 +74,11 @@ AnsiString PathUtil::getDirectoryName(const AnsiString &path) {
 AnsiString PathUtil::getFileName(const AnsiString &path) {
 	AnsiString newPath = unifySeparators(path);
 	Common::String lastPart = Common::lastPathComponent(newPath, '/');
+
+	if (lastPart.empty()) {
+		return path;
+	}
```

Comment:
Given the context (https://github.com/scummvm/scummvm/commit/afee4aeacccf66f52fefacbb0904049f0ce45227#diff-cb293c2a8565f312766de8a207314d1eL90) I wouldn't be surprised if it was a leftover from... something.